calyx-py/calyx/builder_util.py

```diff
@@ -0,0 +1,175 @@
+# pylint: disable=import-error
+import calyx.builder as cb
+
+
+def insert_eq(comp: cb.ComponentBuilder, left, right, cellname, width):
+    """Inserts wiring into component {comp} to check if {left} == {right}.
+    1. Within {comp}, creates a combinational group called {cellname}_group.
+    2. Within the group, creates a cell {cellname} that checks equalities of {width}.
+    3. Puts the values {left} and {right} into the cell.
+    4. Returns the equality-checking cell and the overall group.
```

Comment:
Here's an idea for how to write these docstrings in an even more grokkable way: give a template for the generated Calyx code, as concrete syntax. So for this one, it would be something like:

```
<cellname> = std_eq(<width>);
...
comb group <cellname>_group {
  <cellname>.left = <left>;
  <cellname>.right = <right>;
}
```

That could be a bit easier to understand at a glance than an English list, at least for me?